tools/proofers/PPage.inc

```diff
@@ -260,6 +260,84 @@ class PPage
 
     // -------------------------------------------------------------------------
 
+    function attempt_to_save_as_done($text_data)
+    // This is only an attempt, because a daily page limit might block the save,
+    // or prevent further saves.
+    // If there's a problem, this function does not return to the caller.
+    {
+        global $code_url, $pguser;
+
+        $projectid = $this->lpage->projectid;
+        $round = $this->lpage->round;
+
+        list($saved, $dpl_reached) = $this->attemptSaveAsDone($text_data, $pguser);
+
+        if (!$dpl_reached)
+        {
+            assert($saved);
+            return; // to let the caller do the appropriate normal thing
+        }
+
+        assert($dpl_reached);
+
+        if ($saved)
+        {
+            $title = _("Saved, but at limit");
+            $sentence = sprintf(
+                _("Your page has been saved as 'Done'. However, you have now reached the daily page limit for %s."),
+                $round->id
+            );
+        }
+        else
+        {
+            $this->saveAsInProgress($text_data, $pguser);
+            $title = _("Already at limit");
+            $sentence = sprintf(
+                _("Your page was saved as 'In Progress' rather than 'Done', because you have already reached the daily page limit for %s."),
+                $round->id
+            );
+        }
+
+        slim_header( $title );
+
+        echo "<p>$sentence</p>\n";
+        
+        echo "<p>"
+            . sprintf(
+                _("You will not be allowed to save any more pages in %s until the next server midnight."),
```

Comment:
We've had a request to tweak the wording here to `"You will be able to save more pages in %s after server midnight."`

Reply:
Actually, that adjustment should be made everywhere the "You will not be allowed to ..." wording appears.

Reply:
Done.

Reply:
> Actually, that adjustment should be made everywhere the "You will not be allowed to ..." wording appears.

This is the only place where that wording appears. However, the project page is another place where that wording *could* have appeared, and it's possible you might want the replacement text to be added there. (Currently, it says `<round> limits you to <n> page-saves per day, and you have already reached that limit today`.)